card/mod/core/set/all/references.rb

```diff
@@ -9,24 +9,30 @@
 
 # cards that refer to self
 def referers
-  references_in.map(&:referer_id).map(&Card.method(:fetch)).compact
+  referer_cards_from_references references_in
 end
 
 # cards that include self
-def includers
-  refs = references_in.where(ref_type: "I")
-  refs.map(&:referer_id).map(&Card.method(:fetch)).compact
+def nesters
```

Comment:
main changes:
1. "include" to "nest" in API
2. these card methods return uniq cards.